frontend/src/components/FeeStats/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useEffect } from "react";
+import { Icon, Table } from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+import { get } from "lodash";
+import BigNumber from "bignumber.js";
+
+import { useRedux } from "hooks/useRedux";
+import { SectionCard } from "components/SectionCard";
+import { fetchFeeStatsDataAction } from "ducks/feeStats";
+import "./styles.scss";
+
+export const FeeStats = () => {
+  const { feeStats } = useRedux("feeStats");
+  const dispatch = useDispatch();
+
+  const labels = [
+    { id: "metric", label: "Metric" },
+    { id: "value", label: "value" },
+  ];
+
+  const nameMap = [
+    { id: "ledger_capacity_usage", name: "Capacity Usage" },
+    { id: "max_fee.max", name: "Max Accepted Fee" },
+    { id: "max_fee.min", name: "Min Accepted Fee" },
+    { id: "max_fee.mode", name: "Mode Accepted Fee" },
+    { id: "max_fee.p10", name: "10th Percentile Accepted Fee" },
+    { id: "max_fee.p20", name: "20th Percentile Accepted Fee" },
+    { id: "max_fee.p30", name: "30th Percentile Accepted Fee" },
+    { id: "max_fee.p40", name: "40th Percentile Accepted Fee" },
+    { id: "max_fee.p50", name: "50th Percentile Accepted Fee" },
+    { id: "max_fee.p60", name: "60th Percentile Accepted Fee" },
+    { id: "max_fee.p70", name: "70th Percentile Accepted Fee" },
+    { id: "max_fee.p80", name: "80th Percentile Accepted Fee" },
+    { id: "max_fee.p90", name: "90th Percentile Accepted Fee" },
+    { id: "max_fee.p95", name: "95th Percentile Accepted Fee" },
+    { id: "max_fee.p99", name: "99th Percentile Accepted Fee" },
+  ];
+
+  const getFeeStats = useCallback(() => {
+    dispatch(fetchFeeStatsDataAction());
+  }, [dispatch]);
+
+  useEffect(() => {
+    getFeeStats();
+    setInterval(() => getFeeStats(), 5 * 1000);
+  }, [getFeeStats]);
+
+  const iconStyle = (value: number) => {
```

Comment:
idk if I agree with these arrow icons. The arrows make it seem like it's a value going up or down (like a trend), when in actually we're using them here just to show magnitude (ie anything over 500 is a downward red arrow)

![Screen Shot 2022-07-19 at 3 48 46 PM](https://user-images.githubusercontent.com/30449853/179861176-848a2aa4-bcc3-4b30-9715-20f4598d3435.png)

maybe something to have our UX team look at after this is merged
cc @jcx120

Reply:
I agree with Alec on this one. We should also add a hover label on these to make it more obvious what it's showing. Right now when the cursor hovers over there is nothing.

Reply:
So what do we need to show when the cursor hovers over there?


are we going to do that in this PR?

Reply:
It doesn't need to be done in this PR if it wasn't mentioned in the issue.